src/utils/datetime/validator.ts

```diff
@@ -1,103 +1,109 @@
 import {DateTime} from 'luxon'
 import {DEFAULT_TIME_FORMAT} from 'src/utils/datetime/constants'
 
-const formatterToDateFnsMap = {
+const formatToLuxonMap = {
   [DEFAULT_TIME_FORMAT]: {
-    format: 'yyyy-MM-dd HH:mm:ss',
+    format: 'yyyy-MM-dd H:mm:ss',
```

Comment:
`H:mm` validates 02:00 and 2:00 as true. we can use the `strictCheck` to implement the strictness.